dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature

```diff
@@ -0,0 +1,32 @@
+Feature: HOC Guide Email Conversion Dialog
+
+  @eyes
+  Scenario: View and Close Dialog as signed in Instructor
+    When I open my eyes to test "signedIn"
+    Given I create a teacher named "Coach"
+    And I sign in as "Coach"
+    Then I am on "http://studio.code.org/s/alltheselfpacedplthings/lessons/1/levels/2"
```

Comment:
out of curiousity, why not just use allthethings?

Reply:
I was copying the levels used in the Age Dialog feature file lol but this is a good point- updated